### ISSUE TITLE
Republish

### ISSUE THREAD

**Issue by strugee** [cjcliffe/NoComply#4]
demos.mozilla.org is gone. It'd be a shame if No Comply was gone from the web, too.

**Comment by cjcliffe** [comment]
@strugee Aye; I don't see any reason I couldn't host it in a gh-pages branch here -- I've got updated code for the audio stuff in another demo that I need to port over to fix it; should probably update FOTN as well :)

**Comment by strugee** [comment]
\o/

FWIW FOTN is still online at https://videos-real-origin.cdn.mozilla.net/uploads/mozhacks/flight-of-the-navigator/ - not sure if you want to do anything about that

**Comment by cjcliffe** [comment]
Yeah I'm pretty sure the APIs it uses are all completely deprecated and none of the feeds work anymore though -- need to put it a bit of static content and rework the audio API usage.

I also have a high-resolution version of FOTN here (adds selector to startup) that was never officially published that could be added.

**Comment by cjcliffe** [comment]
Leaving this here for reference; since it will be mostly the same fix: https://github.com/cjcliffe/BeatDetektor3-Demo/commit/3df1444a8604f878fb673c421fb9bc1a52a651cc

**Comment by cjcliffe** [comment]
@strugee No Comply now up and seemingly functional at: http://cjcliffe.github.io/NoComply/

**Comment by strugee** [comment]
@cjcliffe thanks! I'm thrilled.

\o/
